src/Sulu/Bundle/ContactBundle/Resources/js/containers/ContactAccountSelection/ContactAccountSelection.js

```diff
@@ -0,0 +1,158 @@
+        this.handleConfirm(accounts, ContactAccountSelectionStore.accountPrefix);
+    };
+
+    callChange() {
+        const {onChange} = this.props;
+
+        onChange(this.loadedIds);
+    }
+
+    handleRemove = (id: string) => {
+        this.store.remove(id);
+        this.callChange();
+    };
+
+    handleSorted = (oldItemIndex: number, newItemIndex: number) => {
+        this.store.move(oldItemIndex, newItemIndex);
+        this.callChange();
+    };
+
+    render() {
+        const {disabled, value} = this.props;
+
+        return (
+            <Fragment>
+                <MultiItemSelection
+                    disabled={disabled || false}
+                    label={translate('sulu_contact.contact_account_selection_label', {count: value ? value.length : 0})}
+                    leftButton={{
+                        icon: 'su-plus-circle',
+                        onClick: this.handleAddButtonClick,
+                        options: [
+                            {label: translate('sulu_contact.people'), value: 'contacts'},
+                            {label: translate('sulu_contact.organizations'), value: 'accounts'},
+                        ],
+                    }}
+                    loading={this.store.loading}
+                    onItemsSorted={this.handleSorted}
+                >
+                    {this.store.items.map((item, index) => (
+                        <MultiItemSelection.Item
+                            id={item.id}
+                            index={index + 1}
+                            key={item.id}
+                            onRemove={this.handleRemove}
+                        >
+                            <div className={contactAccountSelectionStyles.item}>
+                                {item.fullName || item.name}
+                            </div>
+                        </MultiItemSelection.Item>
+                    ))}
+                </MultiItemSelection>
+                <MultiListOverlay
+                    adapter="table"
+                    listKey="contacts"
+                    onClose={this.handleOverlayClose}
+                    onConfirm={this.handleContactConfirm}
+                    open={this.openedOverlay === 'contacts'}
+                    preloadSelectedItems={false}
+                    preSelectedItems={this.store.contactItems}
+                    resourceKey="contacts"
+                    title={translate('sulu_contact.contact_selection_overlay_title')}
+                />
+                <MultiListOverlay
+                    adapter="table"
+                    listKey="accounts"
+                    onClose={this.handleOverlayClose}
+                    onConfirm={this.handleAccountConfirm}
+                    open={this.openedOverlay === 'accounts'}
+                    preloadSelectedItems={false}
+                    preSelectedItems={this.store.accountItems}
+                    resourceKey="accounts"
```

Comment:
i think we should extract global constants for our resource keys sometime. would be helpful for custom js too. but we dont need to do this now